packages/common/src/data/applicant_stream_data.ts

```diff
@@ -0,0 +1,1057 @@
+export type Stream = Record<
```

Comment:
1. Why file name is in snake case?
2. This file is very big, any thought of refactor this to smaller one?

Reply:
1. No good reason 😄 
2. The data can't really be easily split up and I've just added a couple typings for it, can you think of a way to split it up? Maybe back to a JSON file and a separate TS file for types?

Reply:
Let discuss refactor approach  and feasibility